ui/src/containers/GphlWorkflowParametersDialog.jsx

```diff
@@ -36,8 +36,8 @@ function renderIndexingTable(indexingTable, selected, onSelectRow) {
               key={tdContent}
               data-selected={selected.includes(index) || undefined}
               data-highlight={
-                indexingTable.highlights[index + 1]
-                  ? indexingTable.highlights[index + 1][0]
+                indexingTable.highlights[index]
+                  ? indexingTable.highlights[index][0]
                   : undefined
               }
               onClick={() => onSelectRow(index, tdContent)}
```

Comment:
NB two lines further down, line 45, should the '+ 1' also be removed??

Reply:
sorry about the +1,  I thought your index stating from 1.